introduction.md

```diff
@@ -0,0 +1,2 @@
+Hello my name is Guillaume and i'm a young french developper, and i'm trying
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.